src/otx/algorithms/classification/configs/configuration.yaml

```
@@ -275,6 +275,34 @@ learning_parameters:
       type: UI_RULES
     visible_in_ui: true
     warning: null
+  input_size:
+    affects_outcome_of: TRAINING
+    default_value: Default
+    description:
+      The input size of the given model could be configured to one of the predefined resolutions.
+      Reduced training and inference time could be expected by using smaller input size.
+      Defaults to per-model default resolution.
+    editable: true
+    enum_name: InputSizePreset
+    header: Configure model input size.
+    options:
+      DEFAULT: "Default"
+      _64x64: "64x64"
+      _128x128: "128x128"
+      _256x256: "256x256"
+      _384x384: "384x384"
+      _512x512: "512x512"
+      _768x768: "768x768"
+      _1024x1024: "1024x1024"
+    type: SELECTABLE
+    ui_rules:
+      action: DISABLE_EDITING
+      operator: AND
+      rules: []
+      type: UI_RULES
+    value: Default
+    visible_in_ui: false
+    warning: Small input size may decrease model accuracy at the price of better train and inference speed.
```

```suggestion
    warning: Modifying input size may decrease model performance
```

Some users might use `larger` resolution instead of `smaller`

---

Right. I'll update it.